games-util/springlobby/springlobby-0.265.ebuild

```diff
@@ -0,0 +1,54 @@
+# Copyright 1999-2018 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=6
+
+WX_GTK_VER="3.0"
+inherit cmake-utils gnome2-utils wxwidgets
+
+DESCRIPTION="The official lobby client for SpringRTS community games"
+HOMEPAGE="https://springlobby.info"
+SRC_URI="https://www.springlobby.info/tarballs/${P}.tar.bz2"
+
+LICENSE="GPL-2+"
+SLOT="0"
+KEYWORDS="~amd64 ~x86"
+IUSE="debug +libnotify +nls +sound"
+
+RDEPEND="
+	>=dev-libs/boost-1.35:=
+	dev-libs/openssl:=
+	sound? ( media-libs/alure
```

Comment:
We keep our dependencies sorted, and (USE-)conditional deps below unconditional ones. So please move this down to bottom of the list, and put those two deps on separate lines.